### DISCUSSION TITLE
Some big problems when I play on phone!!! (Chinese+English)

### DISCUSSION THREAD

**Discussion by adtcqd** [HasangerGames/suroi#333] [General]
I came from China so ↓is written by translate APP，if you can read Chinese，pleaseread the Chinese part，thanks you！

English part（Chinese part is in the↓↓↓）：
Hello, I come from China. I often play this game with my mobile phone, and I will also play this game with my classmates, but we encounter many problems! 1. How does the mobile version put items on the ground for teammates to pick up? 2. Although I chose the Asian region, I still have a delay of 100+. However, if I use the https://suroi.fpsgo.net/ game, although it is version 0.15, the delay is very low. Why can't I merge these two servers? 3. The logical error in throwing grenades in the mobile version is not to loosen the rocker and throw. If forced operation, the Grenade may be thrown to the right! Thank you for reading my poor English, which is the limit of my English level:) Leave the rest to the translation software! ! !

中文部分（更加准确 但可能对您造成阅读障碍）：
你好 我来自中国 我常常使用手机玩这个游戏 还会和全班同学一起玩这个游戏 但是我们遇到很多问题！
1.手机版如何把物品放在地上给队友捡起？
2.虽然我选择了亚洲区 但是我依旧有100+的延迟 然而如果我用https://suroi.fpsgo.net/游戏 虽然这是0.15版本 但是延迟很低 请问为什么不能把这两个服务器合并？
3.手机版投掷手雷时操作逻辑错误 并不是松开摇杆投掷 如果强行操作可能导致手雷固定向右投掷！
4.谢谢你阅读我不流畅的英语 这是我英语水平的极限:) 其他的就交给翻译软件吧！！！

**Comment by adtcqd** [comment]
抱歉 我好像重复的发了一遍 因为我所在的地区很难稳定使用GitHub: (
Sorry, I seem to have repeated it because it is difficult to use GitHub stably in my area: (

**Comment by bien-star** [comment]
-Server ping is determined by physical distance from the server. 
-Mobile features like item dropping and better gun/grenade mechanics are in development.

**Reply by adtcqd** [reply]
谢谢 但是请问https://suroi.fpsgo.net 和 suroi.io 有何区别？我在前者可以流畅游玩 但是只有15.0版本 有很多bug

Thank you, but what's the difference between https://suroi.fpsgo.net and suroi.io? I can play smoothly in the former, but only version 15.0 has many bugs.谢谢 但是请问https://suroi.fpsgo.net 和 suroi.io 有何区别？我在前者可以流畅游玩 但是只有15.0版本 有很多bug

Thank you, but what's the difference between https://suroi.fpsgo.net and suroi.io? I can play smoothly in the former, but only version 15.0 has many bugs.
